hunting/hunting.py

```diff
@@ -97,16 +100,10 @@ async def leaderboard(self, ctx, global_leaderboard=False):
         for account in sorted_acc:
             if account[1]["total"] == 0:
                 continue
-            try:
-                if account[0] in [member.id for member in ctx.guild.members]:
-                    user_obj = self.bot.get_user(account[0])
-                else:
-                    if global_leaderboard:
-                        user_obj = account[0]
-                    else:
-                        continue
-            except AttributeError:
-                user_obj = account[0]
+            if global_leaderboard or (account[0] in [member.id for member in ctx.guild.members]):
+                user_obj = self.bot.get_user(account[0]) or account[0]
+            else:
```

Comment:
```py
# is that even possible?
```

Reply:
Not really sure now that we've messed around with this a bit. Good enough though.